api-bucket-replication.go

```diff
@@ -289,3 +289,53 @@ func (c *Client) GetBucketReplicationResyncStatus(ctx context.Context, bucketNam
 	}
 	return rinfo, nil
 }
+
+// BucketReplicationDiff - gets diff for unreplicated entries.
+func (c *Client) BucketReplicationDiff(ctx context.Context, bucketName string, opts replication.DiffOpts) <-chan replication.DiffInfo {
+	diffCh := make(chan replication.DiffInfo)
+
+	// start a routine to start reading line by line.
+	go func(diffCh chan<- replication.DiffInfo) {
+		defer close(diffCh)
+		urlValues := make(url.Values)
+		urlValues.Set("replication-diff", "")
```

Comment:
```suggestion
		urlValues.Set("replication-diff", "")
```

Since the API is for replication for a more terse API this should be just `diff` @poornas

Reply:
Just like arn is not called `replication-arn`

Reply:
api route is like `minio:9001/bucket/?replication-diff=` using just "diff" would be too terse

Reply:
yeah but why it would be just "replication-diff" ? isn't it just `diff` anyways?